benchmark/benchmarkTests.go

```diff
@@ -185,8 +198,11 @@ func OverlayFSFullRun(
 		b.Fatalf("%s", err)
 	}
 	log.G(ctx).WithField("benchmark", "CreateContainer").WithField("event", "Start").Infof("Start Create Container")
+	createContainerStart := time.Now()
 	container, cleanupContainer, err := containerdProcess.CreateContainer(ctx, image)
+	createContainerDuration := time.Since(createContainerStart)
 	log.G(ctx).WithField("benchmark", "CreateContainer").WithField("event", "Stop").Infof("Stop Create Container")
+	b.ReportMetric(float64(createContainerDuration.Milliseconds()), "createContainerDuration")
```

Comment:
Since  `createContainerDuration` is not included in the final stats we should not include it here.

Reply:
Updated